src/shared_modules/utils/archiveHelper.hpp

```diff
@@ -0,0 +1,154 @@
+/*
+ * Utils abstract wait
+ * Copyright (C) 2015, Wazuh Inc.
+ * Feb 8, 2024.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#ifndef _ARCHIVE_HELPER_HPP
+#define _ARCHIVE_HELPER_HPP
+
+#include "archive.h"
+#include "archive_entry.h"
+#include "customDeleter.hpp"
+#include <filesystem>
```

Comment:
Add vector and algorithm includes to maintain the helper atomically.

Reply:
Hi @Dwordcito, I did not understand this comment. Do you mean to avoid using filesystem here?